ci/ci.yaml

```diff
@@ -0,0 +1,217 @@
+---
+resource_types:
+  - name: pull-request
+    type: registry-image
+    source:
+      repository: docker.io/teliaoss/github-pr-resource
+resources:
+  - name: pull-request
+    type: pull-request
+    icon: source-branch
+    webhook_token: ((webhook-token))
+    check_every: 1h
+    source:
+      repository: realliance/senior-matchmaking
+      access_token: ((github-access-token))
+      disable_ci_skip: true
+  - name: senior-matchmaking-main
+    type: git
+    icon: git
+    webhook_token: ((webhook-token))
+    check_every: 1h
+    source:
+      uri: https://github.com/realliance/senior-matchmaking
+      branch: main
+  - name: quay
+    type: registry-image
+    icon: docker
+    check_every: 1h
+    source:
+      repository: quay.io/realliance/senior-matchmaking
+      tag: latest
+      username: ((quay-ci-account.username))
+      password: ((quay-ci-account.token))
+  - name: node
+    type: registry-image
+    icon: docker
+    check_every: 1h
+    source:
+      repository: docker.io/node
+      tag: fermium-alpine
+  - name: python
+    type: registry-image
+    icon: docker
+    check_every: 1h
+    source:
+      repository: docker.io/python
+      tag: alpine3.12
+  - name: oci-build-task
+    type: registry-image
+    icon: docker
+    check_every: 1h
+    source:
+      repository: docker.io/vito/oci-build-task
+  - name: sentry-cli
+    type: registry-image
+    icon: docker
+    check_every: 1h
+    source:
+      repository: docker.io/getsentry/sentry-cli
+jobs:
+  - name: validate-pr
+    plan:
+      - get: pull-request
+        trigger: true
+        version: every
+      - put: pull-request
+        params:
+          path: pull-request
+          status: pending
+      - get: node
+      - get: python
+      - task: install-packages
+        image: node
+        config:
+          platform: linux
+          inputs:
+            - name: pull-request
+              path: .
+          caches:
+            - path: node_modules
+            - path: .yarn
+          run:
+            path: /bin/sh
+            args:
+              - -exc
+              - |
+                yarn install --cache-folder .yarn
```

Comment:
wtf is the point of this?

Reply:
Fixed in 1dabfd76be8637e7a32c37bdd56c731038f0f0f3